.circleci/config.yml

```diff
@@ -148,6 +152,7 @@ jobs:
             python3 -m venv venv
             . venv/bin/activate
             cd python
+            pip install --upgrade pip
```

Comment:
Why was this necessary?

Reply:
The base pip version on the circleci docker image wasn't able to install a package. I forget which one.